Makefile

```diff
@@ -157,7 +157,7 @@ all: build
 .PHONY: help
 help:
 	@echo "Make Routines:"
-	@echo " - \"\"                               equivalent to \"build\""
```

Comment:
We shouldn't remove this tab. If we remove it, the output might be broken.

The outputted texts were already aligned before:

<img width="527" alt="image" src="https://user-images.githubusercontent.com/2114189/147843209-257d0d00-9b74-4f4d-a77b-28d0f9e75b66.png">

Reply:
We could use `'` instead of `"` for this line to make the source code more beautiful.

Reply:
@wxiaoguang reverted this change.